### PR TITLE
Acrolinx quality improvements: cprofile.md

### PR DESCRIPTION
**Pull request by gswashington**

### CODE REVIEW

**Comment by Jak-MS** [comment]
#assign: @robinharwood, @Xelu86 

Can you review the proposed changes? IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge.

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: WindowsServerDocs/administration/windows-commands/cprofile.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
 ---
 title: cprofile
-description: Reference article for the cprofile command. This command has been deprecated and is not guaranteed to be supported in future releases of Windows.
+description: Reference article for the cprofile command. This command is deprecated and is not guaranteed to be supported in future releases of Windows.
```
Comment (robinharwood) [inline WindowsServerDocs/administration/windows-commands/cprofile.md:3]:
```suggestion
description: Reference article for the cprofile command. The command cprofile is deprecated and maybe removed them from a future releases of Windows.
```

File: WindowsServerDocs/administration/windows-commands/cprofile.md [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+ms.date: 01/03/2023
 ---
 
 # cprofile
 
-Cprofile is deprecated and is not guaranteed to be supported in future releases of Windows.
+Applies to: Applies to: Windows Server (All supported versions)
 
-Terminal Services has been updated to Remote Desktop Services. For more information, see [Welcome to Remote Desktop Services](../../remote/remote-desktop-services/welcome-to-rds.md).
+The command cprofile is deprecated and is not guaranteed to be supported in future releases of Windows.
```
Comment (robinharwood) [inline WindowsServerDocs/administration/windows-commands/cprofile.md:16]:
```suggestion
The command cprofile is deprecated and maybe removed them from a future releases of Windows.
```

**Review by robinharwood — CHANGES_REQUESTED** [review]
Thanks @gswashington - just a couple of changes.

Thanks,
Robin

**Comment by robinharwood** [comment]
As mentioned, I'm closing this PR down.

**Pull request closed without merging**